src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -89,6 +89,17 @@ $header-height: 45px;
     &__second-icon {
       margin-left: 5px;
```

Comment:
How to deal with the redundant class and spacing literal:
https://github.com/sparkletown/sparkle/pull/2007/files#diff-977ce0dba740bcc42e948ea6052d729b246d573391d10f3a95fb9c1a7448df98L89-R97